pkg/sdn/plugin/node_iptables.go

```diff
@@ -91,11 +91,10 @@ func (n *NodeIPTables) syncIPTableRules() error {
 // Get openshift iptables rules
 func (n *NodeIPTables) getStaticNodeIPTablesRules() []FirewallRule {
 	return []FirewallRule{
-		{"nat", "POSTROUTING", []string{"-s", n.clusterNetworkCIDR, "-j", "MASQUERADE"}},
-		{"filter", "INPUT", []string{"-p", "udp", "-m", "multiport", "--dports", VXLAN_PORT, "-m", "comment", "--comment", "001 vxlan incoming", "-j", "ACCEPT"}},
-		{"filter", "INPUT", []string{"-i", TUN, "-m", "comment", "--comment", "traffic from SDN", "-j", "ACCEPT"}},
-		{"filter", "INPUT", []string{"-i", "docker0", "-m", "comment", "--comment", "traffic from docker", "-j", "ACCEPT"}},
```

Comment:
Huh. Not only did we add it for a reason, but I'm the one who added it, and it was fairly recently. Oops. (See #11919; we need it to allow traffic from docker0 to the node itself, eg for DNS.)